### DISCUSSION TITLE
Please HELP to add my SSH key to the SSH agent

### DISCUSSION THREAD

**Discussion by fideltova** [community/community#91826] [Codespaces]
### Select Topic Area

General

### Body

Hi guys, I'm signed in to the CS50 database course, but can't send my tests for review because when I try to add my SSH key to the agent the following error shows up:

Warning: unknown mime-type for /home/ubuntu/.ssh/config -- using application/octet-stream Error: no view mailcap rules found for type application/octet-stream

NOTE: I'm not into programming, I've just started my database course but this is very difficult for me since it's not my field. 

Thanks!

**Comment by fideltova** [comment]
I've followed these steps but it seems they are not working: https://cs50.readthedocs.io/submit50/

**Reply by krabelize** [reply]
Hi @fideltova, try to copy the SSH key to the on the server side to the authorized_keys file:

`vi /home/$USER/.ssh/authorized_keys`
